README.md

```diff
@@ -1,129 +1,170 @@
+
+For more information, see our [documentation](https://controls-docs.readthedocs.io/en/latest/).
+
+**[Back to top](#table-of-contents)**
+
+## Project Status
+
+The Controls System is currently on release **M2.0** for the 2022-24 build cycle, to compete in FSGP 2024. See our [Versioning](#versioning).
+
+**[Back to top](#table-of-contents)**
+
+## Getting Started
+
+The preferred development environment for the system is VSCode on a Linux machine. Options for developers on Windows environments include WSL or a Virtual Machine. Instructions provided here will assume that you have a working Linux environment with terminal control.
+
+### Workspace Setup
+
+1. Clone this repository via [SSH](https://docs.github.com/en/authentication/connecting-to-github-with-ssh): ```git clone --recurse-submodules git@github.com:lhr-solar/Controls.git```.
+
+1. In VSCode, remote into your Linux machine if needed. Click **File** &#8594; **Open Workspace from File** and select the workspace located in ```.vscode/LHR.code-workspace```.
+
+1. Run ```Embedded-Sharepoint/Scripts/install.sh``` to selectively install your packages. For Controls development, you will need the following packages:
+    * ARM Toolchain
+    * build-essential
+    * gdb-multiarch
+    * openocd
+    * stlink-tools
+    * Renode Simulator
+    * Documentation Tools (Sphinx, Doxygen, and Breathe)
+
+    The script will prompt for installation of each of these packages.
+
+1. Add ```source /path/to/Controls/Scripts/load_env_vars.sh``` to the end of your ```~/.bashrc``` file in order to load the necessary environment variables on shell startup. A typical example: ```source /${HOME}/LHR/Controls/Scripts/load_env_vars.sh```
+
+1. Recommended extensions for VSCode:
+    * C/C++ Extension Pack
+    * GitHub Pull Requests
+    * GitLens
+    * Subway Surfers
+
+### Build System
+
+We use the GNU Make utility as our build system.
+
+Examples:
+```
+make leader
+make leader TEST=HelloWorld
+make leader TEST=HelloWorld DEBUG=1
+make flash
+make docs
+```
+
+### Running Tests
+
+WIP: A formal test framework has yet to be defined for the Controls system. For now, ```make leader TEST=TestName``` should build the Controls system excluding **main.c** and including **Tests/Test_TestName.c**
+
+### Formatting
+
+WIP: clang-format and clang-tidy are in the process of being integrated into our workflow.
+
+### Usage
+
+Follow our documentation to set up the Controls test rig, power the board with 12V, and use ```make flash``` to flash the board with your ```Objects/controls-leader.elf``` executable, either built from source or placed in the Objects folder from your intended release.
+
+For debugging purposes, make sure there is a Nucleo plugged into your computer and connected to the leaderboard via JTAG. Run ```openocd``` in one bash session and ```gdb-multiarch``` in another, then attach to the process openocd has started.
```

Comment:
You should clarify this more. A new person reading this would not be able to understand. You also didn't include the target-extended remote... command that has to be run in gdb. I think the BPS readme has a good example.

Reply:
Ok, I'm pulling and modifying some stuff from the BPS readme.